### PR TITLE
LPS-202228 Change Configuration view for Blogs

### PR DESCRIPTION
**Pull request by BarbaraCabrera**
## Motivation

[Link to ticket](https://liferay.atlassian.net/browse/LPS-202228)

## Changes

Change the configuration for Blogs because it should be open in a new page instead of a modal.

## Notes

**You need to activate the LPS-197692 FF.**

## Test Scenario 1

1. Go to Blogs
2. Click on the kebab menu in the top right
3. Navigate to the sections to check the behaviour and styles

https://github.com/liferay-echo/liferay-portal/assets/93203423/53111ec4-ccc4-43ad-a77b-ccdd690b47ed

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by BarbaraCabrera** [comment]
SF failures not related to the changes in this PR. _AutoSF_ commit added to fix it.

**Comment by BarbaraCabrera** [comment]
ci:test:sf

**Comment by BarbaraCabrera** [comment]
ci:test:sf

File: modules/apps/blogs/blogs-web/src/main/java/com/liferay/blogs/web/internal/display/context/BlogsConfigurationDisplayContext.java [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+	}
+
+	public VerticalNavItemList getSettingsVerticalNavItemList() {
+		return VerticalNavItemListBuilder.add(
+			_getVerticalNavItemUnsafeConsumer("rss")
+		).build();
+	}
+
+	public String getSubtitle() {
+		if (Objects.equals(getNavigation(), "structures")) {
```
Comment (jkappler) [inline modules/apps/blogs/blogs-web/src/main/java/com/liferay/blogs/web/internal/display/context/BlogsConfigurationDisplayContext.java:123]: @BarbaraCabrera, do we have structures, and highlighted in blogs?

File: modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs_admin/configuration_browser.jsp [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+							<clay:content-col>
+								<%= blogsConfigurationDisplayContext.getSubtitle() %>
+							</clay:content-col>
+						</clay:content-row>
+					</h2>
+
+					<aui:input name="<%= Constants.CMD %>" type="hidden" value="<%= Constants.UPDATE %>" />
+					<aui:input name="redirect" type="hidden" value="<%= blogsConfigurationDisplayContext.getRedirect() %>" />
+
+					<liferay-ui:error key="emailFromAddress" message="please-enter-a-valid-email-address" />
```
Comment (jkappler) [inline modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs_admin/configuration_browser.jsp:77]: EmailFromAddress and emailFromName shouldn't be required, you already have validators at input level @BarbaraCabrera

File: modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs_admin/configuration_browser.jsp [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+									delta="<%= GetterUtil.getInteger(blogsGroupServiceOverriddenConfiguration.rssDelta()) %>"
+									displayStyle="<%= blogsGroupServiceOverriddenConfiguration.rssDisplayStyle() %>"
+									enabled="<%= blogsGroupServiceOverriddenConfiguration.enableRss() %>"
+									feedType="<%= blogsGroupServiceOverriddenConfiguration.rssFeedType() %>"
+								/>
+							</div>
+						</c:when>
+						<c:otherwise>
+							<div class="c-px-4">
+								<aui:input cssClass="lfr-input-text-container" label="name" name="preferences--emailFromName--" type="text" value="<%= blogsGroupServiceSettings.getEmailFromName() %>">
```
Comment (jkappler) [inline modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs_admin/configuration_browser.jsp:159]: @BarbaraCabrera, these validators don't seem to be working

 
Barbara

**Comment by jkappler** [comment]
@BarbaraCabrera, can you please fix the commits? It's a bit weird to have "autoSF", "methods" and  "jsp reorganize". I think it will be tidier after you include them as part of the changes.

Thanks 🙏

**Comment by jkappler** [comment]
Hi @BarbaraCabrera, all works correctly, just have two comments. 

Shouldn't the first selected item be RSS? It's weird that I navigate to the second option which are emails. If not, we should move RSS to the bottom.
 
Barbara

It makes no sense to have the definitions where we define the user and email address. 

Screenshot 2023-11-23 at 12 52 19

All of this is most probably a bad design

**Comment by jkappler** [comment]
@BarbaraCabrera, why do we have "email" as title in RSS?

 
Screenshot 2023-11-23 at 12 56 36

**Comment by jkappler** [comment]
@BarbaraCabrera, sorry that I missed it as I mentioned, but we can hide RSS, so we have to take this into account

**Comment by jkappler** [comment]
https://github.com/liferay-echo/liferay-portal/pull/14493

**Pull request closed without merging**